tests/nsrt_learning/strips_learning/test_gen_to_spec_learner.py

```diff
@@ -86,17 +90,19 @@ def test_backchaining_strips_learner():
 
     # Test sidelining where an existing operator needs to
     # be specialized.
-    goal3 = {Asleep([bob]), Sad([bob])}
```

Comment:
just curious what's the motivation behind the changes to this existing test?

Reply:
that test turned out to be wrong in that it was relying on the bug (i.e, not checking datastore length change) to pass. I modified the input so that it would produce the same output given our fix.